src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react';
+import { fetchPeople } from '../utils/fetchPeople';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+
+const BASE_URL
+  = 'https://mate-academy.github.io/react_people-table/api/people.json';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchPeople(BASE_URL)
+      .then(setPeople)
+      .catch(() => setIsError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+
+          {isLoading
+            ? <Loader />
+            : (
+              <>
+                {isError && (
+                  <p data-cy="peopleLoadingError" className="has-text-danger">
+                    Something went wrong
+                  </p>
+                )}
+
+                {people.length
```

Comment:
About errors: our page should look like on the first screenshot, when ```network offline```. In your website I see smth like that (look on the second screenshot). But I see that your tests passed, but I don't know why (hahah), because my tests thought that is wrong variant

![image](https://github.com/mate-academy/react_people-table-basics/assets/142230575/bbd497d9-07ec-4dda-b3b2-19f9630de81f)


![image](https://github.com/mate-academy/react_people-table-basics/assets/142230575/92f1fb5d-4b07-42a8-b5d0-76a342103c27)